model.py

```diff
@@ -95,9 +84,9 @@ class Block(nn.Module):
 
     def __init__(self, config):
         super().__init__()
-        self.ln_1 = LayerNorm(config.n_embd, bias=config.bias)
+        self.ln_1 = nn.LayerNorm(config.n_embd, bias=config.bias)
```

Comment:
Nice. To add another confirmation, I wrote my own model and used `nn.LayerNorm` directly with no issues: https://github.com/vhmth/gpt-2/blob/main/model.py#L74